spec/factories/miq_server.rb

```diff
@@ -7,5 +7,9 @@
     started_on      { Time.now.utc }
     stopped_on      { "" }
     version         { '9.9.9.9' }
+
+    factory :miq_server_in_default_zone do
+      zone          { FactoryBot.build(:zone, :name => "default") }
```

Comment:
This was to get around errors in test where non-default zone servers created in 'podified' scenarios having 2 servers per zone due to this [odd validation](https://github.com/ManageIQ/manageiq/blob/acdc1ccdaa3a01060415f2499455e3e3fe9b54ca/app/models/zone.rb#L254-L259)

Reply:
Basically, the default `:miq_server` factory in with `is_podified?` stubbed `true` causes the first server to be created with a zone and that zone creation causes a second server to be created in that zone, thereby doubling the expected number of servers.